src/send-bsd.h

```diff
@@ -23,23 +24,47 @@
 #ifdef ZMAP_SEND_LINUX_H
 #error "Don't include both send-bsd.h and send-linux.h"
 #endif
-#ifdef ZMAP_SEND_MAC_H
-#error "Don't include both send-bsd.h and send-mac.h"
-#endif
-
 
 int send_run_init(UNUSED sock_t sock)
 {
 	// Don't need to do anything on BSD-like variants
 	return EXIT_SUCCESS;
 }
 
-int send_packet(sock_t sock, void *buf, int len, UNUSED uint32_t idx)
+int send_packet(sock_t sock, void *buf, int len, UNUSED uint32_t retry_ct)
```

Comment:
```suggestion
int send_packet(sock_t sock, void *buf, int len, uint32_t retry_ct)
```
IMO we can remove this flag, since at least in some compilation cases this field _is_ used. WDYT?

Reply:
The purpose of the `UNUSED` is to silence the compiler warning in the compilation case(s) where the argument is _not_ used.  Keeping the `UNUSED` makes sense, unless we use the argument in _all_ compilation cases.

Reply:
Fair! Sounds good, resolving.